bindings/python/pymongocrypt/explicit_encrypter.py

```diff
@@ -30,9 +31,15 @@ def __init__(self, algorithm, key_id=None, key_alt_name=None,
           - `query_type` (str): The query type to execute.
           - `contention_factor` (int): The contention factor to use
             when the algorithm is "Indexed".
+          - `range_options` (bytes): Options for explicit encryption
```

Comment:
I feel like a longer name like `query_range_options` would be better because it only works when you have `query_type="rangePreview"` according to the spec.

Reply:
I prefer not to make this name longer. Longer names are usually worse and these apis are already very wordy. https://journal.stuffwithstuff.com/2016/06/16/long-names-are-long/ is a great blog post on the subject of naming.

In this case I'd say the "range" in "range_opts" already qualifies this as only pertaining to rangePreview queries. The C driver uses "rangeopts" but we usually add an underscore between words so I shortened the name to "range_opts".